apps/design-system/config/docs.ts

```diff
@@ -117,6 +117,11 @@ export const docsConfig: DocsConfig = {
           href: '/docs/fragments/inner-side-menu',
           items: [],
         },
+        {
+          title: 'Data Input',
+          href: '/docs/fragments/data-input',
+          items: [],
+        },
```

Comment:
I’ve documented this component under _Fragments_ since:

1. That’s what it is.
2. We already have (confusingly) an Input component under the Components category.

We should obviously consolidate, merge, or clarify the difference. But one step at a time.